### ISSUE TITLE
Export view design and content differs from what is displayed in the builder

### ISSUE THREAD

**Issue by CSwartzHMA** [18F/cms-hitech-apd#2222]
During content review, we identified some areas where the export view is inconsistent with the builder content.

### This task is done when...
 - [x] Areas that differ in design or content have been identified here for discussion
 - [x] Cause of view discrepancies has been identified and addressed
 - [x] Design and content is in alignment across views
 - [x] A documentation of how we approach the different versions is agreed on and added to the wiki 
 - [x] The findings from this issue are broken out into useable content, design and dev issues

**Comment by CSwartzHMA** [comment]
(1) Export view reorders and changes content of Executive Summary and order of milestones (chronological by end date).
(2) Export view seems to omit Allocation section in Cost Allocation and Budget for FFY display, but shows math for 90/10, 75/25, etc.

**Comment by CSwartzHMA** [comment]
Executive Summary in builder:
[screenshot]

In Export view:
[screenshot]

**Comment by CSwartzHMA** [comment]
Relating to item 2: Field headings are present, but content is not carried over in export view. The cost amount does not match the builder. Summary appears to be pulling from the wrong activity:

Builder View:
![image](https://user-images.githubusercontent.com/45497254/82845216-a9990800-9eb1-11ea-8285-2f39b25c12d0.png)

Export View: 
![image](https://user-images.githubusercontent.com/45497254/82845412-73a85380-9eb2-11ea-912b-1c808c22e6b0.png)

Builder Budget:
![image](https://user-images.githubusercontent.com/45497254/82845632-4c9e5180-9eb3-11ea-8919-283c254efebd.png)

Export Budget:
![image](https://user-images.githubusercontent.com/45497254/82845650-5a53d700-9eb3-11ea-8b8c-d4931b9e12a7.png)

Builder Totals:
![image](https://user-images.githubusercontent.com/45497254/82845681-7ce5f000-9eb3-11ea-9007-b32c19bbea34.png)


Export Totals:
![image](https://user-images.githubusercontent.com/45497254/82845701-91c28380-9eb3-11ea-9fe0-93e902ed8a3f.png)

**Comment by CSwartzHMA** [comment]
Other inconsistencies:
Activity List:  Page Heading is "Program Activities" in builder and "List of Planned Activities" in Export.
Activity Overview: Heading is "Provider a short overview of activity." in builder and "Activity Summary:" in export.
Standards and Conditions is displayed at the very end of the activity information in Export, but is paired with the overview in the builder.
State Cost Categories: Builder page title is "State Staff", but Export header is "In-House Cost Categories: State Personnel".  Builder page title is "Other state expenses", but Export Header is "In-House Cost Categories: Non-Personnel".

**Comment by nicholeweems** [comment]
For future discussion and review by @pphillips-fearless. Potential design issues to be spun up upon review and discussion. Option to create Epic as well based upon decision.

**Comment by beparticular** [comment]
Next steps: Jenn is going to take a look at this as a bigger picture discussion of the builder + the export view + the executive summary and look at how we work on a solid process around making sure we take these two pieces of our app in consideration when we design and/or iterate.

**Comment by beparticular** [comment]
Notes from design sync to be cleaned up and added to wiki to document process around the different versions. 

Executive summary 
-We only need Activity Summary and how much it costs (Act Sum and Cost) 
-If you make a change in Activities, make sure it doesn’t affect, or is changed in the Executive Summary and in the export view
- Nothing else is changed
- Put executive summary on the update experience roadmap for things to be considered. 

Export view
- Issue exists (this one) - make sure fields all exist and are in the right order, document what exists in the export version that doesn’t exist in the builder and why, is it necessary. 
- Future - design needs to make sure we account for the export view, fields in the same order, if something looks weird doing that, address thru design vs a change in field order. 

Builder view
- If a change in activity, verify exec summary and export view
- Verify export view

Error messages - if custom, need to be changed if field label changes

**Comment by jjames521** [comment]
Just for posterities sake and ordering, I went through the entire app again after Caryn and found the same issues.  I also attempted to put the issues in order for ease of use.

Results of Previous Activities Page - Where is Prior Activities Overview? Headings differ. Suggest updating Export to match what is displayed in the builder.  Actual costs heading is Actual Costs of Previous Activities.

Activity List: Page Heading is "Program Activities" in builder and "List of Planned Activities" in Export.

Activity Overview: Heading is "Provider a short overview of activity." in builder and "Activity Summary:" in export.

Standards and Conditions is displayed at the very end of the activity information in Export, but is paired with the overview in the builder.

State Cost Categories: Builder page title is "State Staff", but Export header is "In-House Cost Categories: State Personnel". Builder page title is "Other state expenses", but Export Header is "In-House Cost Categories: Non-Personnel”.

OKR Page: Favor export view

Other Funding Amount: The cost amount does not match the builder.  Summary appears to be pulling from the wrong activity.

Federal State Split: 50/50 for example not rolling into export view

FFY 2020-2021 Totals

Private Contractor Cost Page formatting/order 

1 Contract Term
2 total cost
3 FFY 2020
4 FFY 2021

1 Contract Term
3 2020 Costs
4 2021 Costs 
2 Total Cost

Proposed Budget: Builder Heading is "Estimated Quarterly Incentive Payments", but Export view says "Incentive Payments".
